app/models/billing/usage/product_catalog.rb

```diff
@@ -8,7 +8,7 @@ def self.all_products
   end
 
   def current_products
-    products = parent.team.billing_subscriptions.active.map(&:included_prices).flatten.map(&:price).map(&:product)
+    products = non_zero_included_prices.map(&:price).compact.map(&:product).compact
```

Comment:
I added these compacts to prevent breakages in the event of a dev or someone removing a product or price from the products.yml